README.md

```diff
@@ -23,6 +23,7 @@ docker run
     -e COORDINATES=-8,13 // optional, this will overwrite COUNTRY env var
     -e TRANSPORTATION='driving+ferry' //optional
     -e TRAVEL_TIME=7200 //optional
+    -e UNIQUE_REQUESTS=1 // optional, this specifies how many unique requests to generate
```

Comment:
I still don't understand. 1 unique request would mean only 1 requests in total or what?

Reply:
yes, so I asked to create a percentage of unique requests

Reply:
Yeah it just states how many requests for it to generate, and then after that it will use that array to send requests.

I'll change it to be a percentage, but it should probably be kept pretty low, `minutes*rpm` can become a large number very easily.